Atomic/sign.py

```diff
@@ -72,6 +72,9 @@ def no_reg_no_default_error(image, registry_path):
                     if signature_path is None:
                         raise ValueError("No write path for {}/{} was "
                                          "found in {}".format(reg, repo, registry_config_path))
+                    elif signature_path.startswith("http"):
+                        raise ValueError("Writing to {} is not supported. Use a "
+                                         "file:///location instead.".format(signature_path))
 
                     # Deal with write path prepends
                     if signature_path.startswith("file://"):
```

Comment:
This is still pretty brittle; what if I say `ftp:`? This should only accept known URL schemas, not randomly look for specific prefixes with `startswith`.